angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-no-results.html

```diff
@@ -0,0 +1,35 @@
+<div class="mp-no-results">
+	<div class="mp-no-results-container">
+		<img class="mp-sad-bucky" src="img/bucky-sad.png">
```

Comment:
I do love the sad bucky, how can we make this good for other schools also? Maybe only display this for madison, but others just see some text unless they have a sad mascot?